docs/user/manifests/kubeapps-local-dev-webhook-rbac.yaml

```diff
@@ -0,0 +1,35 @@
+# Retrieve the token by executing:
+#   kubectl get secret $(kubectl get serviceaccount apprepositories-webhook -o jsonpath='{range .secrets[*]}{.name}{"\n"}{end}' | grep apprepositories-webhook) -o jsonpath='{.data.token}' -o go-template='{{.data.token | base64decode}}' && echo
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: apprepositories-webhook
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: apprepositories-webhook
+rules:
+  - apiGroups:
+      - kubeapps.com
+    resources:
+      - apprepositories
+    verbs:
+      - get
+      - update
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: apprepositories-webhook
+  namespace: kubeapps
```

Comment:
I actually forgot to mention it in the docs. Thank you for bringing it to my attention!

But, yes, I was thinking about what's might be the best approach for the example. Whereas using the kubeapps namespace will work seamlessly for users who just deploy the default kubeapps chart with the Bitnami repo, it has, as you mention, security concerns. Given that the general docs also include the cluster-admin for the sake of simplicity, I opted to exemplify using the non-namespaced repo.

Nevertheless, since the target user using webhooks may be the one who is installing a new custom repo (i.e., Harbor), I believe we should choose the namespaced repo approach in the docs. I'll modify the docs accordingly.